src/app/dataset/workflow/workflow-header/workflow-header.component.spec.ts

```diff
@@ -14,6 +14,22 @@ describe('WorkflowHeaderComponent', () => {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   let formGroupConf = {} as any;
 
+  const getEvent = (): { dragDT: DragDT; eventDragDT: EventDragDT } => {
+    const dragDT = ({
+      setData(s: string, v: string): void {
+        console.log(s, v);
+      },
+      setDragImage(el: HTMLElement, left: number, top: number): void {
+        console.log(el, left, top);
+      }
```

Comment:
This `setData` and `setDragImage` seem a bit useless. Where are these used?

Reply:
In the workflow header, on start drag, the functions are necessary to set the drag data and the orb.  In this test suite simulcra of the element functions are needed to satisy the compiler but also to spy upon (see lines #194 / #195) of the same file.